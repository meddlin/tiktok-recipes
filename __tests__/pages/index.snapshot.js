import { render } from '@testing-library/react';
import Home from '../../pages/index';

it('renders homepage unchanged', () => {
    const sampleSearchArray = [];
    const { container } = render(<Home searchArray={sampleSearchArray} />);

  expect(container).toMatchSnapshot()
})