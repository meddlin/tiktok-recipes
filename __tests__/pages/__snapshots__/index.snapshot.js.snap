// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="flex flex-col h-screen justify-between items-center"
  >
    <main>
      <h1
        class="text-2xl font-bold underline"
      >
         Recipes 
      </h1>
      <input
        placeholder="Search first..."
      />
      No results yet
    </main>
    <footer
      class=""
    >
      <p
        class="text-xl"
      >
        Recipes collection curated from TikTok
      </p>
    </footer>
  </div>
</div>
`;
